apps/dashboard/src/components/activity/activity-header.tsx

```diff
@@ -0,0 +1,17 @@
+import { RiRouteFill } from 'react-icons/ri';
+
+import { cn } from '@/utils/ui';
+
+export const ActivityHeader = ({ title, className }: { title?: string; className?: string }) => {
```

Comment:
I've split the `ActivityPanel` into smaller composable components, this is one of them